internal/provider/resource_account.go

```diff
@@ -91,6 +88,7 @@ func setFromAccountResponseMap(d *schema.ResourceData, raw map[string]interface{
 		return err
 	}
 
+	// TODO(malnick) - remove after deprecation cycle in favor of attributes
 	switch raw["type"].(string) {
 	case accountTypePassword:
```

Comment:
looks like there's no way to change the password?

Reply:
This function sets from the response, which never returns a password. But more to the point, shouldn't we _always_ set attributes if they are returned, regardless of what type of account it is?

Reply:
Resolving for now as dealing with arbitrary attributes is not yet on the table